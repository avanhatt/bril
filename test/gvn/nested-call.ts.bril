test/ts/nested-call.ts.bril