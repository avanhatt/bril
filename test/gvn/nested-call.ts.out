void main() {
b1:
  v01: int = const 1;
  v11: int = call getTwo();
  v21: int = add v01 v11;
  print v21;
  ret ;
}
int getTwo() {
b1:
  ret v01;
}
